manifests/server/passenger.pp

```diff
@@ -57,10 +60,16 @@
   ]
 
   if $puppet_ca_proxy and $puppet_ca_proxy != '' {
+    puppet::server::combine_certs { $ssl_combined:
```

Comment:
Because the passenger setup is deprecated (IIRC Puppet 6 will remove it), I'd prefer to inline the resources here. It's not many